pynets/stats/embeddings.py

```diff
@@ -552,43 +589,62 @@ def build_omnetome(est_path_iterlist, ID):
         est_path_iterlist = [est_path_iterlist]
 
     if len(est_path_iterlist) > 1:
-        atlases = list(set([x.split("/")[-3].split("/")[0]
-                            for x in est_path_iterlist]))
-        parcel_dict_func = dict.fromkeys(atlases)
-        parcel_dict_dwi = dict.fromkeys(atlases)
-
-        est_path_iterlist_dwi = list(
-            set(
-                [
-                    i
-                    for i in est_path_iterlist
-                    if i.split("model-")[1].split("_")[0] in struct_models
-                ]
+        try: #For standard file formatting
+            atlases = list(set([x.split("/")[-3].split("/")[0]
+                                for x in est_path_iterlist]))
+            parcel_dict_func = dict.fromkeys(atlases)
+            parcel_dict_dwi = dict.fromkeys(atlases)
+
+            est_path_iterlist_dwi = list(
+                set(
+                    [
+                        i
+                        for i in est_path_iterlist
+                        if i.split("model-")[1].split("_")[0] in struct_models
+                    ]
+                )
             )
-        )
-        est_path_iterlist_func = list(
-            set(
-                [
-                    i
-                    for i in est_path_iterlist
-                    if i.split("model-")[1].split("_")[0] in func_models
-                ]
+            est_path_iterlist_func = list(
+                set(
+                    [
+                        i
+                        for i in est_path_iterlist
+                        if i.split("model-")[1].split("_")[0] in func_models
+                    ]
+                )
             )
-        )
 
-        if "_rsn" in ";".join(est_path_iterlist_func):
-            func_subnets = list(
-                set([i.split("_rsn-")[1].split("_")[0] for i in
-                     est_path_iterlist_func])
-            )
-        else:
+            if "_rsn" in ";".join(est_path_iterlist_func):
+                func_subnets = list(
+                    set([i.split("_rsn-")[1].split("_")[0] for i in
+                        est_path_iterlist_func])
+                )
+            else:
+                func_subnets = []
+            if "_rsn" in ";".join(est_path_iterlist_dwi):
+                dwi_subnets = list(
+                    set([i.split("_rsn-")[1].split("_")[0] for i in
+                        est_path_iterlist_dwi])
+                )
+            else:
+                dwi_subnets = []
+        except: #For nonstandard file formatting
+            print(
+                "WARNING: Non-standard file naming detected. "
+                "Using default values for embedding."
+                )
+
+            nonstandard_bool = True
```

Comment:
Oops, I forgot to ask you about this- since build_omnetome uses graph names to determine how to process them, I went ahead and included all graphs in both the dwi and func lists to try to cover the different ways _omni_embed can process them, but I didn't make it run any of the graphs as subnets because I was unsure if/how I could make it analyze these correctly from random data. I meant to ask you about the difference between dwi and func and whether its worth analyzing each random/unnamed graph as both, or if it would be better to analyze all of the graphs as simply one or the other. Also, if the subnet graphs don't have to be mathematically related to one another other than by their shape, we could also do something like analyze all random graphs like they're subsets of one big graph if you think that'd be worth doing.

Also, 637 - 649 looked to me like it would be useful both when the program has standard file formatting and when it doesn't, but to keep from duplicating the code I thought I needed to keep it out of the try except that happened right before it. I then ran into a problem on how to differentiate from standard and nonstandard file formatting later on, since if I didn't the function passed empty lists to _omni_embed with the nonstandard option. Because of this, I created the boolean to help with this. Another option we could potentially do though that I think would let us get rid of the boolean would be to go ahead and extend the try except to cover the if-else beginning on line 653, and make extra lines of code to cover what's being done in lines 637 - 649 in both the try and except parts of the try-except block if that'd be better.

I completely forgot to bring these thing up and ask you about them- sorry about that!